ace/os_linux.go

```diff
@@ -0,0 +1,63 @@
+// +build linux
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+func checkMountImpl(d string, readonly bool) error {
+	mountinfoPath := fmt.Sprintf("/proc/self/mountinfo")
+	mi, err := os.Open(mountinfoPath)
+	if err != nil {
+		return err
+	}
+	defer mi.Close()
+
+	sc := bufio.NewScanner(mi)
+	for sc.Scan() {
+		var (
+			mountID        int
+			parentID       int
+			majorMinor     string
+			root           string
+			mountPoint     string
+			mountOptions   string
+			optionalFields string
```

Comment:
according to `proc(5)`:
> (7)  optional fields: zero or more fields of the form "tag[:value]".

Does the `Scanf` work if there are several fields? It would be good to add some unit tests about that.